polkadot/scripts/list-syscalls/execute-worker-syscalls

```diff
@@ -0,0 +1,71 @@
+0 (read)
+1 (write)
+2 (open)
+3 (close)
+4 (stat)
+5 (fstat)
+7 (poll)
+8 (lseek)
+9 (mmap)
+10 (mprotect)
+11 (munmap)
+12 (brk)
+13 (rt_sigaction)
+14 (rt_sigprocmask)
+15 (rt_sigreturn)
+16 (ioctl)
+19 (readv)
+20 (writev)
+24 (sched_yield)
+25 (mremap)
+28 (madvise)
+39 (getpid)
+41 (socket)
+42 (connect)
+45 (recvfrom)
+46 (sendmsg)
+53 (socketpair)
+55 (getsockopt)
+56 (clone)
```

Comment:
I see particularly dangerous syscalls here, like `clone`

Is there a possibility of further tailoring this list to the ones required strictly during PVF execution, after all the wasmtime setup is done? We only really need to install the filter right before executing the PVF, not during the prerequisite setup, which should enable us to tighten this list

Reply:
I had an idea of having a "blacklist for the whitelist", that is, pick out all the I/O syscalls from this list and additionally block those. Makes sense to implement that at the same time as logging, and we'll see if any I/O calls get through the filter. For now, as we discussed, I just want some reasonable assurance that `io_uring` is never called, so we can properly block networking in the interim. :)